src/oasis-nexus/api.ts

```diff
@@ -822,6 +827,31 @@ export const useGetConsensusProposals: typeof generated.useGetConsensusProposals
   })
 }
 
+export const useGetConsensusProposalsProposalId: typeof generated.useGetConsensusProposalsProposalId = (
+  network,
+  proposalId,
+  options?,
+) => {
+  return generated.useGetConsensusProposalsProposalId(network, proposalId, {
+    ...options,
+    request: {
+      ...options?.request,
+      transformResponse: [
+        ...arrayify(axios.defaults.transformResponse),
+        (data: generated.Proposal, headers, status) => {
+          if (status !== 200) return data
+          return {
+            ...data,
+            network,
```

Comment:
https://github.com/oasisprotocol/explorer/pull/1192#discussion_r1470516266

Reply:
Yes, adding the consensus layer here now.